### PR TITLE
add dualstack support

### LINKED ISSUE #182: Ipv6 support
I was trying to setup an IPv6 endpoint with the created ALB but I do not find any reference to it.

Is it supported or do you have any plan to support it?

### PR DESCRIPTION
**Pull request by szuecs**
fix #182 
Signed-off-by: Sandor Szücs <sandor.szuecs@zalando.de>

### CODE REVIEW

File: controller.go [hunk 1]
```diff
@@ -92,6 +93,7 @@ func loadSettings() error {
 		fmt.Sprintf("sets the maximum number of certificates to be attached to an ALB. Cannot be higher than %d", aws.DefaultMaxCertsPerALB))
 	flag.StringVar(&sslPolicy, "ssl-policy", aws.DefaultSslPolicy, "Security policy that will define the protocols/ciphers accepts by the SSL listener")
 	flag.StringVar(&blacklistCertARN, "blacklist-certificate-arns", "", "Certificate ARNs to not consider by the controller: arn1,arn2,..")
+	flag.StringVar(&ipAddressType, "ip-addr-type", aws.DefaultIpAddressType, "IPAdress type to use, one of 'ipv4' or 'dualstack'")
```
Comment (mikkeloscar) [inline controller.go]: nit: `IPAdress` -> `IP Address`

File: aws/cf.go [hunk 2]
```diff
@@ -148,6 +150,7 @@ func createStack(svc cloudformationiface.CloudFormationAPI, spec *stackSpec) (st
 			cfParam(parameterTargetGroupVPCIDParameter, spec.vpcID),
 			cfParam(parameterTargetTargetPortParameter, fmt.Sprintf("%d", spec.targetPort)),
 			cfParam(parameterListenerSslPolicyParameter, spec.sslPolicy),
+			cfParam(parameterIpAddressTypeParameter, spec.ipAddressType),
```
Comment (mikkeloscar) [inline aws/cf.go]: You also need to define this parameter in the `aws/cf_template.go` and reference it on the LB resource.

Comment (szuecs) [reply aws/cf.go]: fixed

**Comment by mikkeloscar** [comment]
:+1:

**Comment by szuecs** [comment]
:+1:

**Pull request merged**